app/assets/javascripts/application.js

```diff
@@ -13,16 +13,18 @@
 //= require angular-ui-bootstrap
 //= require angular-ui-bootstrap-tpls
 //= require angular-sanitize
+//= require angular-ui-router
 //= require moment
 //= require moment-strftime/build/moment-strftime.min
 //= require moment-timezone
 //= require sprintf
 //= require numeral
 //= require miq_api
 //= require miq_angular_application
+//= require_tree ./components/
```

Comment:
..and if you move those two to directives, we can remove this line, so.. this whole file should probably stay unchanged?